src/BayesFilters/src/GPFCorrection.cpp

```diff
@@ -144,7 +145,5 @@ double GPFCorrection::evaluateProposal
 {
     /* Evaluate the proposal distribution, a Gaussian centered in 'mean' and having
        covariance 'covariance', in the state 'state'. */
-    VectorXd difference = state - mean;
-
-    return (-0.5 * static_cast<double>(difference.size()) * log(2.0 * M_PI) -0.5 * log(covariance.determinant()) -0.5 * (difference.transpose() * covariance.inverse() * difference).array()).exp().coeff(0);
+    return utils::multivariate_gaussian_density(state, mean, covariance).coeff(0);
```

Comment:
This use to return the log likelihood so this change is wrong. Once implemented, use `multivariate_gaussian_log_density`.

Reply:
Maybe there is a misunderstanding. Since in this function we are returning the standard Gaussian density, there is a trailing `.exp().coeff(0)`, are you proposing to use
```
std::exp(multivariate_gaussian_log_density(...))
```
?

Reply:
My bad, that's ok so 👍
This conversation and similar ones are resolved.